### ISSUE TITLE
Server's IP address not found

### ISSUE THREAD

**Issue by PAC23pacman** [steveseguin/social_stream#156]
Hi,
today my obs chat ninja plugin is not loading. any help? my internet is working fine.
Obs 25.0.1 on windows 7
thanks
![2023-11-11_10h07_54](https://github.com/steveseguin/social_stream/assets/145875405/8ed08eb6-9ced-411e-9508-a58acefe2147)

**Comment by steveseguin** [comment]
Social Stream is hosted by Github pages, so if its not accessible, it might be a issue with Github itself or your DNS route to it. Seems to be working fine for me, however I can look into having it hosted in a secondary place, for backup.

It currently is hosted already locally though, so you can access it locally from the extension's folder directly.  For example: 
```
file:///C:/Users/steve/Code/social_stream/dock.html?session=yKFXm55dr6
```

![image](https://github.com/steveseguin/social_stream/assets/2575698/ec2d7535-c4e9-4014-af3e-7a702be32d06)


![image](https://github.com/steveseguin/social_stream/assets/2575698/ae6a287c-4feb-4137-8d98-92a18828c0fa)

**Comment by steveseguin** [comment]
When specifically were you having issues?

I looked at Github's incident reports and it mentions it had problems last night for a short while.
![image](https://github.com/steveseguin/social_stream/assets/2575698/d58db3e7-aab1-4ed7-a898-586bd4c26a9d)

**Comment by astral0pitek** [comment]
Hello. When I enabling local server it shows me this address and I don't know how to reach this file.

<img width="423" height="113" alt="Image" src="https://github.com/user-attachments/assets/ebacf0e1-397c-4beb-a0e9-991ace1fa974" />

**Comment by steveseguin** [comment]
thank you.

I'll look into it, although I'm not replicating the issue at the moment

<img width="671" height="457" alt="Image" src="https://github.com/user-attachments/assets/acd7fa85-8af7-4a67-9ecb-080eafb00bf0" />

<img width="1091" height="639" alt="Image" src="https://github.com/user-attachments/assets/56ac8d66-1105-4bde-b4b8-e50d655c2527" />

<img width="632" height="690" alt="Image" src="https://github.com/user-attachments/assets/2999a066-a50a-4468-af2e-cf7dd5bf1da0" />

**Comment by astral0pitek** [comment]
I also have this error, so maybe these 2 problems are related.
Sorry for not showing this error earlier, I didn't notice it.

<img width="1137" height="239" alt="Image" src="https://github.com/user-attachments/assets/5a29be11-e50a-49c1-bb16-5f2d71786da2" />

<img width="1268" height="795" alt="Image" src="https://github.com/user-attachments/assets/5f95ceae-babb-41b8-a766-7d1d7959c8da" />

**Comment by steveseguin** [comment]
Hi @astral0pitek , it sounds a bit like it's not able to download the assess from Github?


One thought I have at the moment is that you might be in a country where global network access is restricted, like Russia.

I'll see if I can improve things to support you better in the next release

**Comment by astral0pitek** [comment]
Yes, you're completely right. I live in Russia. Before I already had problems with blocked access to assets and the socialstream.ninja website before, but that was resolved by blocking type HTTPS of DNS requests. But now it seems things are getting really bad, and I'd be happy if I could continue using your app.
